### DISCUSSION TITLE
Change "Top Creators All Time" to "Top Active Creators"

### DISCUSSION THREAD

**Discussion by arhebbar** [deso-protocol/dips#138] [Medium CIPs]
This is a follow-on, on a [recent post ](https://bitclout.com/posts/9b8fc425b4dd92afea69516ef2d037e677225db7fd2a7eb6c252173db8697bda?tab=posts) of mine.

I had posted about this a few weeks ago and since it didn't get picked up, I am posting this here.

Suggestion #1:
- Change "Top Creators All Time" to "Top Active Creators - Last 7 Days" or "Top Weekly Creators"
- Filter the Creators with at least one post or one (transaction= follow, like, comment, etc.) within the last 7 days
- Default to this for new users

Suggestion #2: 
- Add Top Weekly Creators to the Trends Page. I see that this doesn't come up in the Trends Page (not sure why). Adding this in the Trends page will ensure that we are incentivizing those who are more active in the last 7 days.
- Add a highlight (*New) or something to highlight that the list has been updated and new Creators have joined the list.

Suggestion #3:  
- Actually show a list by Top Users in terms of Activity (Likes & Diamonds would be a fair proxy) 
- Especially if the last 7 days filter is added, this would really incentivize people to focus on creating content here instead of exiting the platform. It would highlight more of the real creators of the platform and the connectors too (@RajLahoti @Krassenstein @MarioNawfal @HuntSauce @DesignSta @AgustinOlmedo and @RickandMortyOG) would get into the list.

Pros:
- Reward and Motivate Active Creators
- Shift money from Inactive Creators to more Active Creators
- Motivate those who are not very active to become more active as they see that activity gets recognition and rewards.

Cons:
- I can't think of any cons. But, I understand Elon Musk and Naval might either be backing this project in stealth mode and/or their likelihood of joining the platform might go down by a small %. But, I think the Pros far outweigh this con.

**Comment by georgeziozas** [comment]
Τhis is a topic that is currently being discussed on the platform. Almost everyone agrees in this direction, yet the core dev team doesn't seem to care about it at this point. NFT's and totally random ideas seem to be more appealing to the devs. Let's see. Great analysis though @arhebbar !

**Reply by arhebbar** [reply]
Thanks! Just to add - @ElonMusk got over 100 trades in the last 24 hours. We all know how many trades us active users have got in the last one week. ;-)

If in doubt, do this change for one day and see how the # of trades dips.

**Comment by amoghgopadi** [comment]
Absolutely!!!! A much required change on Bitclout

**Comment by arhebbar** [comment]
@maebeam @diamondhands0 @lazynina @redpartyhat @bluepartyhat - This is the most upvoted discussion in < 24 hours and I don't think it's that difficult to implement. Is there a reason we are still sticking to the reserved "ElonMusk" and "Naval" ids at the top and misleading many new investors to go invest in them? If there is, I'd love to know.

**Comment by arhebbar** [comment]
Adding on to this, I did some work on the DB that I got access to thanks to @Kanshi and Love4Src and pulled some names and made a post tagging those most active in the last 5-7 days (post the $CLOUT dip). The reaction to the 2-3 posts tells you what the community thinks. 

https://bitclout.com/posts/f8c99005c0e62d5561fbf0659bb0eb4eb528bf051273df79c0ce480b816c4e3a?tab=posts

https://bitclout.com/posts/34df6783dfc482c4cfbbcf2a71273d9f92426969306ff36b4e5eaf31b1c1faa8?tab=posts

I think the onus is on the Core Team and the Nodes to pick this up. For now, it might be even as simple as running the query on the DB and updating the table once a day.
